aggregator/persist.go

```diff
@@ -101,6 +103,7 @@ func (a *Aggregator) processSvc(d k8s.K8sResourceMessage) {
 		Type:       string(service.Spec.Type),
 		ClusterIPs: service.Spec.ClusterIPs,
 		Ports:      ports,
+		Selector:   service.Spec.Selector,
```

Comment:
I thought it might be useful to persist the service's label selector as well, but I have not yet found a use for this.